.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,84 @@
+name: Build Docker container and deploy app to AWS
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  workflow_dispatch:
+
+env:
+  ECR_REPOSITORY: alex-ic
+  REGION: ap-southeast-1
+
+jobs:
+  build_and_deploy:
+    name: Build the Docker image
```

Comment:
The name here is misleading as it indicates there is just the build step but actually the step does more than that
```suggestion
    name: Build the Docker image and deploy to AWS
```
<img width="694" alt="Screenshot 2023-02-01 at 12 17 49" src="https://user-images.githubusercontent.com/1896814/215957780-82e33142-6b2d-405c-a2fa-7de23d76bb78.png">